### PR TITLE
Various improvements to Ionide article

### LINKED ISSUE #5605: ClassLibraryWhat?
You start by telling us to create "ClassLibrarySample", then everything else refers to "ClassLibraryDemo". It's easy enough to make the adjustment and still follow the tutorial steps, though. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c543c271-0800-4ff9-4dfb-5154efb85732
* Version Independent ID: f1a4752f-a001-5e5e-2604-358db6f5adbc
* Content: [Get Started with F# in Visual Studio Code](https://docs.microsoft.com/en-us/dotnet/fsharp/get-started/get-started-vscode?tabs=windows)
* Content Source: [docs/fsharp/get-started/get-started-vscode.md](https://github.com/dotnet/docs/blob/master/docs/fsharp/get-started/get-started-vscode.md)
* Service: **unspecified**
* Product: **dotnet-fsharp**
* GitHub Login: @cartermp
* Microsoft Alias: **phcart**

### PR DESCRIPTION
**Pull request by cartermp**
Fixes #5605

Also:

* Removes some superfluous images to help with load times for less fast internet connections
* Adds a tab control for using the command palette
* Adds more formatting for code
* Simplifies a few explanations

### CODE REVIEW

File: docs/fsharp/get-started/get-started-vscode.md [hunk 1]
```diff
@@ -42,58 +42,50 @@ You must also install the [.NET Core SDK](https://www.microsoft.com/net/download
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: should you change ms.date?

not related to your change: line 8 - change order and typos
IDE (Integrade Development Enivronment) -> Integrated Development Environment (IDE)

I'd also run Acrolinx on this article to make sure there are no further updates needed. A lot of my suggestions would be caught by that.

File: docs/fsharp/get-started/get-started-vscode.md [hunk 2]
```diff
@@ -42,58 +42,50 @@ You must also install the [.NET Core SDK](https://www.microsoft.com/net/download
...
-1. Use the Command Palette (Ctrl+Shift+P on Windows, ⌘+Shift+P on macOS, Ctrl+Shift+P on Linux) and type the following:
-
-    ```
-    >ext install Ionide
-    ```
-
-2. Click the Extensions icon and search for "Ionide":
-
-    ![](media/getting-started-vscode/vscode-ext.png)
+Next, click the Extensions icon and search for "Ionide":
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: next line: you can also install Ionide-FAKE and to get FAKE support -> you can also install Ionide-FAKE to get FAKE support

File: docs/fsharp/get-started/get-started-vscode.md [hunk 3]
```diff
@@ -42,58 +42,50 @@ You must also install the [.NET Core SDK](https://www.microsoft.com/net/download
...
 
 ## Creating your first project with Ionide
 
 To create a new F# project, open Visual Studio Code in a new folder (you can name it whatever you like).
 
-![](media/getting-started-vscode/vscode-open-dir.png)
+### [Windows](#tab/windows)
 
-Next, open the Command Palette (Ctrl+Shift+P on Windows, ⌘+Shift+P on macOS, Ctrl+Shift+P on Linux) and type the following:
+Next, open the Command Palette (Ctrl+Shift+P) and type the following:
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: you don't need the tabs if you use the menu instead View > Command Palette

File: docs/fsharp/get-started/get-started-vscode.md [hunk 4]
```diff
@@ -42,58 +42,50 @@ You must also install the [.NET Core SDK](https://www.microsoft.com/net/download
...
+---
+
+This is powered by the [FORGE](https://github.com/fsharp-editing/Forge) project. You should see something like this:
 
-This lets you create the project in a different directory, if you like.  Leave it blank for now.  It will create the project in the current directory.  Once you press **Enter**, you will reach the following step:
+> [!NOTE]
+If you don't see template options, try refreshing templates by running the following command in the Command Palette: `>F#: Refresh Project Templates`.
 
-![](media/getting-started-vscode/vscode-proj-name.png)
+Select "F#: New Project" by hitting **Enter**, which will take you to the next step, which allows you to select project types.
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: which will take you to the next step, which allows you to select project types -> which takes you to the next step, that allows you to select project types

File: docs/fsharp/get-started/get-started-vscode.md [hunk 5]
```diff
@@ -42,58 +42,50 @@ You must also install the [.NET Core SDK](https://www.microsoft.com/net/download
...
 
-This lets you name your project.  F# uses [Pascal case](http://c2.com/cgi/wiki?PascalCase) for project names.  This article uses `ClassLibraryDemo` as the name.  Once you've entered the name you want for your project, hit **Enter**.
+Pick the `classlib` template and hit **Enter**.
 
-If you followed the previous step steps, you should get the Visual Studio Code Workspace on the left-hand side to look something like this:
+Next, pick a directory to create the project in. If you leave it blank, it will use the current directory.
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: will use -> uses

File: docs/fsharp/get-started/get-started-vscode.md [hunk 6]
```diff
@@ -42,58 +42,50 @@ You must also install the [.NET Core SDK](https://www.microsoft.com/net/download
...
-    ```
-    >ext install Ionide
-    ```
-
-2. Click the Extensions icon and search for "Ionide":
-
-    ![](media/getting-started-vscode/vscode-ext.png)
+Next, click the Extensions icon and search for "Ionide":
 
 The only plugin required for F# support in Visual Studio Code is [Ionide-fsharp](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-fsharp). However, you can also install [Ionide-FAKE](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-FAKE) and to get [FAKE](https://fsharp.github.io/FAKE/) support and [Ionide-Paket](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-Paket) to get [Paket](https://fsprojects.github.io/Paket/) support. FAKE and Paket are additonal F# community tools for building projects and managing dependencies, respectively.
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: typo: additonal

File: docs/fsharp/get-started/get-started-vscode.md [hunk 7]
```diff
@@ -42,58 +42,50 @@ You must also install the [.NET Core SDK](https://www.microsoft.com/net/download
...
+Pick the `classlib` template and hit **Enter**.
 
-If you followed the previous step steps, you should get the Visual Studio Code Workspace on the left-hand side to look something like this:
+Next, pick a directory to create the project in. If you leave it blank, it will use the current directory.
 
-![](media/getting-started-vscode/vscode-new-proj-explorer.png)
+Finally, name your project in the final step. F# uses [Pascal case](http://c2.com/cgi/wiki?PascalCase) for project names. This article uses `ClassLibraryDemo` as the name. Once you've entered the name you want for your project, hit **Enter**.
 
-This template generates a few things you'll find useful:
+If you followed the previous step steps, you should get the Visual Studio Code Workspace on the left-hand side to appear with the following:
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: step steps

File: docs/fsharp/get-started/get-started-vscode.md [hunk 8]
```diff
@@ -42,58 +42,50 @@ You must also install the [.NET Core SDK](https://www.microsoft.com/net/download
...
 
 1. The F# project itself, underneath the `ClassLibraryDemo` folder.
 2. The correct directory structure for adding packages via [`Paket`](https://fsprojects.github.io/Paket/).
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: step 4: The `paket.exe` executable which -> The `paket.exe` executable that. Same change on steps 3 and 4 of the next section

File: docs/fsharp/get-started/get-started-vscode.md [hunk 9]
```diff
@@ -189,7 +181,7 @@ There's one final thing to notice about this: there's no explicit instruction to
 
 ## Moving your script code into the implementation file
 
-The previous sections in this article demonstrated a common first step in writing F# code: writing an initial function and executing it interactively with FSI.  This is known as REPL-driven development, where REPL stands for "Read-Evaluate-Print Loop".  It's a great way to experiment with functionality until you have something working.
+The previous sections in this article demonstrated a common first step in writing F# code: writing an initial function and executing it interactively with FSI.  This is known as REPL-driven development, where [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop) stands for "Read-Evaluate-Print Loop".  It's a great way to experiment with functionality until you have something working.
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: remove extra spaces between sentences

**Review by mairaw — CHANGES_REQUESTED** [review]
Changes look good but I think this topic can be further improved. Left some comments.

File: docs/fsharp/get-started/get-started-vscode.md [hunk 10]
```diff
@@ -46,51 +46,37 @@ You can install Visual Studio Code from the [code.visualstudio.com](https://code
...
-Next, open the Command Palette (⌘+Shift+P) and type the following:
-
->F# new project
-
-### [Linux](#tab/linux)
-
-Next, open the Command Palette (Ctrl+Shift+P) and type the following:
-
->F# new project
+Next, open the command pallette (**View > Command Pallette) and type the following:
```
Comment (pkulikov) [inline docs/fsharp/get-started/get-started-vscode.md]: Close bold formatting: `(**View > Command Pallette) and...` should be `(**View > Command Pallette**) and...`

File: docs/fsharp/get-started/get-started-vscode.md (hunk 1 above)
Comment (cartermp) [reply docs/fsharp/get-started/get-started-vscode.md]: Changed ms.date and this suggestion. I can run acrolinx today as well.

**Review by mairaw — APPROVED** [review]
I did one small change on the last commit but this is good to 🚢.

**Comment by cartermp** [comment]
Thanks @mairaw!

**Pull request merged**